client/src/components/Checkbox.css

```diff
@@ -1,20 +1,46 @@
 .checkbox {
   display: flex;
   align-items: center;
-  cursor: pointer;
+}
+
+.checkbox--label-before {
```

Comment:
What is the need for label-before? Why not just have every checkbox have the label before since that is the only thing in the designs.

Reply:
anders and i worked together on this PR, so as far as fealty to designs go, we should be in safe territory. anders noted that despite his designs, the implementation in place was such that the label showed up on the right of the checkbox. rather than potentially break the existing code, he felt it made sense to add a variant that it might be easier to make the component useful in more places, especially as more functionality is added.

Reply:
That is fine.

Reply:
As far as matching designs I don't really care other than that the label should extend to the full width.